README.md

```diff
@@ -53,6 +54,34 @@ Run the following commands to prepare the environment and management cluster:
 
 Apply customizations provided by this repo:
 
+we provide two ways to apply customizations,i.e GitOps and manually.
+
+1. Bootstrap AgoCD instance on management Cluster 
+```bash
+export gitops_repo=https://github.com/<your_orgOrName>/declarative-openshift.git #<your newly created repo>
+export cluster_name=managementCluster #<your management cluster name >
+export cluster_base_domain=$(oc get ingress.config.openshift.io cluster --template={{.spec.domain}} | sed -e "s/^apps.//")
+export platform_base_domain=${cluster_base_domain#*.}
+oc apply -f .bootstrap/subscription.yaml
+oc apply -f .bootstrap/cluster-rolebinding.yaml
+envsubst < .bootstrap/argocd.yaml | oc apply -f -
+ ```
+
+2. Bootstrap management Cluster confiurations
+
+replace values in .bootstrap/rosa-hcp-application/values.yaml
+
+```bash
+helm template .bootstrap/rosa-hcp-application   -f .bootstrap/rosa-hcp-application/values.yaml | oc apply -f -
+```
+
+bootstrap rosa-hcp-application will create below resources
+**_Optional_**: openshift-management step is needed if your mangement cluster is an OpenShift Cluster
+![customized managementCluster](./pics/managementCluster.png)
+
+
+2. or Apply customization manually
+
 **_Optional_**: This step is needed if your mangement cluster is an OpenShift Cluster
 
 ```
```

Comment:
1. It would be handy if we could add a line of documentation to update the value of  `targetRevision: integrations-v1` in .bootstrap/integration-application/values.yaml  & .bootstrap/rosa-hcp-application/values.yaml 
2. At the end, we can add a heading on CrossPlane and link it to its readme file.

Reply:
fixed